### PR TITLE
Grammar and Tidelift link updates

### PR DESCRIPTION
**Pull request by esivakumar18**
## Checklist

 - [X] Read the [contributing guide](https://github.com/mockito/mockito/blob/main/.github/CONTRIBUTING.md)
 - [X] PR should be motivated, i.e. what does it fix, why, and if relevant how
 - [X] If possible / relevant include an example in the description, that could help all readers
       including project members to get a better picture of the change
 - [X] Avoid other runtime dependencies
 - [X] Meaningful commit history ; intention is important please rebase your commit history so that each
       commit is meaningful and help the people that will explore a change in 2 years
 - [X] The pull request follows coding style
 - [X] Mention `Fixes #<issue number>` in the description _if relevant_
 - [X] At least one commit should mention `Fixes #<issue number>` _if relevant_

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
      alt="Mockito" />
 </a>
 
-Most popular mocking framework for Java
+Most popular mocking framework for Java.
```
Comment (TimvdLippe) [inline README.md:7]: Nit: let's leave the dot here, as this is our tagline

**Review by TimvdLippe — CHANGES_REQUESTED** [review]
1 nit, other than that LGTM

File: README.md (hunk 1 above)
Comment (esivakumar18) [reply README.md]: Thank you @TimvdLippe, its resolved now.

**Review by TimvdLippe — APPROVED** [review]
Thanks!

**Pull request merged**